action.yml

```diff
@@ -36,290 +33,29 @@ inputs:
-    value: ${{ steps.stack-path.outputs.snapshot-doc-root }}
-  local-doc-root:
-    description: "`local-doc-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-doc-root }}
-  local-hoogle-root:
-    description: "`local-hoogle-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-hoogle-root }}
-  stack-root:
-    description: "`stack-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.stack-root }}
-  project-root:
-    description: "`project-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.project-root }}
-  config-location:
-    description: "`config-location` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.config-location }}
-  bin-path:
-    description: "`bin-path` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.bin-path }}
-  programs:
-    description: "`programs` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.programs }}
-  compiler-exe:
-    description: "`compiler-exe` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.compiler-exe }}
-  compiler-bin:
-    description: "`compiler-bin` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.compiler-bin }}
-  compiler-tools-bin:
-    description: "`compiler-tools-bin` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.compiler-tools-bin }}
-  local-bin:
-    description: "`local-bin` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-bin }}
-  extra-include-dirs:
-    description: "`extra-include-dirs` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.extra-include-dirs }}
-  extra-library-dirs:
-    description: "`extra-library-dirs` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.extra-library-dirs }}
-  snapshot-pkg-db:
-    description: "`snapshot-pkg-db` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.snapshot-pkg-db }}
-  local-pkg-db:
-    description: "`local-pkg-db` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-pkg-db }}
-  global-pkg-db:
-    description: "`global-pkg-db` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.global-pkg-db }}
-  ghc-package-path:
-    description: "`ghc-package-path` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.ghc-package-path }}
-  snapshot-install-root:
-    description: "`snapshot-install-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.snapshot-install-root }}
-  local-install-root:
-    description: "`local-install-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-install-root }}
-  dist-dir:
-    description: "`dist-dir` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.dist-dir }}
-  local-hpc-root:
-    description: "`local-hpc-root` value from `stack path`"
-    value: ${{ steps.stack-path.outputs.local-hpc-root }}
+  no-upgrade-stack:
```

Comment:
Prefer `upgrade-stack` with `default: true`. Makes later logic read a little clear ("if upgrade-stack..." versus "if not no-upgrade-stack...") and matches other inputs better. I forget what this convention is called.

Reply:
Sure will do. I had it this way because I was going to not specify a `default` and let the "default" behavior for booleans flow, but it seems that's not possible as the implicit default is `""` and GitHub refuses to cast it to boolean, apparently.